src/main/java/com/auth0/jwk/JwkSet.java

```diff
@@ -0,0 +1,78 @@
+package com.auth0.jwk;
+
+import java.util.Collections;
+import java.util.List;
+import static com.google.common.base.Preconditions.checkArgument;
+
+/**
+ * Represents a set of JSON Web Keys (JWK) used to verify the signature of JWTs.
+ */
+public class JwkSet {
```

Comment:
If this PR alone is not adding logic to use this class, I suggest you keep it package private for now until you tie everything together